course7/homeworks/patrickkostjens/complex/complex.cc

```diff
@@ -0,0 +1,133 @@
+#include <cmath>
+#include <iostream>
+#include <stack>
+#include <stdlib.h>
+#include <string>
+#include <regex>
+
+struct Complex
+{
+	double real = 0.0;
+	double imaginary = 0.0;
+
+	Complex() = default;
+	Complex(double r) : real(r) {}
+	Complex(double r, double i) : real(r), imaginary(i) {}
+
+	Complex(const Complex& other) : real(other.real), imaginary(other.imaginary) {}
```

Comment:
Do you need to implement this? Can you just use the implementation provided by the compiler? (applies also to ```operator=```)